src/Razor/Microsoft.AspNetCore.Razor.Tools/src/Microsoft.AspNetCore.Razor.Tools.csproj

```diff
@@ -37,7 +37,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json" Version="$(Razor_NewtonsoftJsonPackageVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(Razor_NewtonsoftJsonPackageVersion)" AllowExplicitReference="true" />
```

Comment:
@NTaylorMullen @pranavkm @Pilchie the project currently builds against NewtonSoft.Json 12.0.2, not 9.0.1 in the 'master' and 'release/5.0-preview8' branches. Will that break customers who upgrade to the P8 SDK❔ If yes, I'll copy this one-liner into a PR for the p8 branch.

Reply:
This is a tool and is not referenced by user projects.

Reply:
Then why are there really strong warnings in https://github.com/dotnet/aspnetcore/blob/6cef541d5fa9162af8a25a8d95639b392d2fd259/eng/Versions.props#L265-L267 ❔  If that's irrelevant, I could just remove the `$(Razor_NewtonsoftJsonPackageVersion)` property and this (one and only) attempted use.

Reply:
@NTaylorMullen @JunTaoLuo what's the history here i.e. why was it considered necessary❔ @JunTaoLuo brought the comments over from aspnetcore-tooling but don't see anything beyond https://github.com/dotnet/aspnetcore/pull/21581/files#r428789251 about the block I quoted.

Reply:
I'm not sure of the historical reason why this was done. i just duplicated the logic and comment from tooling. Maybe @pranavkm or @ajaybhargavb can explain further.

Reply:
/ping @pranavkm and @NTaylorMullen is `$(Razor_NewtonsoftJsonPackageVersion)` still relevant❔ I'd rather remove the exception than make the current change if it'll work.

Reply:
Those warnings are from the tooling time where our other tooling based binaries require a host bound newtonsoft.json version. This tool as pranav said is not referenced by user projects and can have whatever version we feel fit.